samples/nrf9160/serial_lte_modem/src/slm_at_host.h

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#ifndef SLM_AT_HOST_
+#define SLM_AT_HOST_
+
+/**@file slm_at_host.h
+ *
+ * @brief AT host for serial LTE modem
+ * @{
+ */
+
+#include <stdlib.h>
+#include <zephyr/types.h>
+
+#include <at_cmd_parser/at_cmd_parser.h>
+#include <modem_info.h>
+
+#define CHECK_ERROR(_x)	if (_x < 0) return _x;
+
+/**@brief AT command callback type. */
+typedef void (* slm_at_callback_t) (char *str);
+
+/**@brief AT command handler type. */
+typedef int (* slm_at_handler_t) (const char *at_cmd_param);
+
+typedef struct slm_at_cmd_list {
+	u8_t type;
+	char *string_upper;
+	char *string_lower;
+	slm_at_handler_t handler;
+} slm_at_cmd_list_t;
+
+/**
+ * @brief Initialize AT host for serial LTE modem
+ *
+ * @retval 0 If the operation was successful.
+ *           Otherwise, a (negative) error code is returned.
+ */
+int at_host_init(void);
```

Comment:
We already have this API in NCS. If the sample needs to reimplement this functionality, please use different name (or just prefix with `slm_` as other APIs here.

Reply:
Updated to slm_at_host_init